### PR TITLE
[stable/cerebro] Enables use of Cerebro provided auth

### PR DESCRIPTION
**Pull request by wvidana**
#### What this PR does / why we need it:
 * Enables use of Cerebro provided auth by using the app's `auth` section in the config file https://github.com/lmenezes/cerebro/blob/f2ed031fd89258c6ad85d40d846f5ff1f610fe73/conf/application.conf#L25-L62
 * Also provides the ability to reference a `Secret` object to get the env vars from.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/cerebro/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: cerebro
-version: 1.0.2
+version: 1.1.2
```
Comment (davidkarlsen) [inline stable/cerebro/Chart.yaml]:
```suggestion
version: 1.1.0
```

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Comment by wvidana** [comment]
@davidkarlsen changes pushed per suggestions https://github.com/helm/charts/pull/14506#pullrequestreview-245752033

**Comment by wvidana** [comment]
@davidkarlsen ping

**Review by davidkarlsen — APPROVED** [review]

**Pull request merged**